packages/chat-headless/src/slices/conversation.ts

```diff
@@ -2,7 +2,11 @@ import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { ConversationState } from "../models/slices/conversation";
 import { Message, MessageNotes } from "@yext/chat-core";
 
-export const initialState: ConversationState = {
+export const STATE_SESSION_STORAGE_KEY : string = "conversationState";
+export const initialState: ConversationState =
+    sessionStorage.getItem(STATE_SESSION_STORAGE_KEY)
+    ? JSON.parse(sessionStorage.getItem(STATE_SESSION_STORAGE_KEY)!)
```

Comment:
would recommend avoiding `!`. In this case, it's probably best to make a helper function to get state in session storage. checking if sessionStorage is defined, and if sessionStorage.getItem(STATE_SESSION_STORAGE_KEY) is defined. Some validation for the value would be good too